setup.py

```diff
@@ -6,41 +6,43 @@
     open('README.rst').read(),
     open('CREDITS.rst').read(),
     open('CHANGES.rst').read(),
-    ])
+])
 
 install_requires = [
-    'Django >= 1.4, < 1.7',
+    'Django >= 1.4',
     'django-extensions',
     'django-nose',
     'requests',
     'itsdangerous',
-    'south',
     'mock',
-    ],
+],
 
-tests_require = [
-    ]
+tests_require = []
 
-setup(name='lizard-auth-client',
-      version=version,
-      description="A client for lizard-auth-server",
-      long_description=long_description,
-      # Get strings from http://www.python.org/pypi?%3Aaction=list_classifiers
-      classifiers=['Programming Language :: Python',
-                   'Framework :: Django',
-                   ],
-      keywords=[],
-      author='Erik-Jan Vos, Remco Gerlich',
-      author_email='remco.gerlich@nelen-schuurmans.nl',
-      url='http://www.nelen-schuurmans.nl/',
-      license='MIT',
-      packages=['lizard_auth_client'],
-      include_package_data=True,
-      zip_safe=False,
-      install_requires=install_requires,
-      tests_require=tests_require,
-      extras_require={'test': tests_require},
-      entry_points={
-          'console_scripts': [
-          ]},
-      )
+setup(
+    name='lizard-auth-client',
+    version=version,
+    description="A client for lizard-auth-server",
+    long_description=long_description,
+    # Get strings from http://www.python.org/pypi?%3Aaction=list_classifiers
+    classifiers=['Programming Language :: Python',
+                 'Framework :: Django',
+                 ],
+    keywords=[],
+    author='Erik-Jan Vos, Remco Gerlich',
+    author_email='remco.gerlich@nelen-schuurmans.nl',
+    url='http://www.nelen-schuurmans.nl/',
+    license='MIT',
+    packages=['lizard_auth_client'],
+    include_package_data=True,
```

Comment:
Ik heb hem eruit gehaald. De tests gaan nu ook goed, dus deze pull-request is klaar om gereviewed te worden.